### PR TITLE
Update err message

### PR DESCRIPTION
**Pull request by ceci3**
update following op error message:
1. batch_norm
2. instance_norm
3. dropout

### CODE REVIEW

File: paddle/fluid/operators/batch_norm_op.cc [hunk 1]
```diff
@@ -60,9 +60,12 @@ void BatchNormOp::InferShape(framework::InferShapeContext *ctx) const {
 
   if (ctx->IsRuntime() && ctx->HasInput("MomentumTensor")) {
     auto mom = ctx->Inputs("MomentumTensor");
-    PADDLE_ENFORCE_EQ(mom.size(), 1,
-                      platform::errors::InvalidArgument(
-                          "Input(MomentumTensor) size must be 1"));
+    PADDLE_ENFORCE_EQ(
+        mom.size(), 1,
+        platform::errors::InvalidArgument(
+            "Input(MomentumTensor) size must be 1"
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cc]: Input(MomentumTensor) ... 改成 
The input tensor MomentumTensor's size must be 1. But received: MomentumTensor's size = [%d]

File: paddle/fluid/operators/batch_norm_op.cc [hunk 2]
```diff
@@ -300,10 +303,14 @@ class BatchNormKernel<platform::CPUDeviceContext, T>
     const auto &x_dims = x->dims();
     PADDLE_ENFORCE_GE(x_dims.size(), 2,
                       platform::errors::InvalidArgument(
-                          "The Input X dim size should be larger than 1."));
+                          "The Input X dimension size should be larger than 1."
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cc]: ‘Input’为什么要大写？
这么写是不是更合适？
The size of input X's dimensions should be larger than 1.

File: paddle/fluid/operators/batch_norm_op.cu [hunk 3]
```diff
@@ -151,10 +154,29 @@ class BatchNormKernel<platform::CUDADeviceContext, T>
...
-      PADDLE_ENFORCE_EQ(est_var->dims()[0], C);
+      PADDLE_ENFORCE_EQ(est_mean->dims().size(), 1UL,
+          platform::errors::InvalidArgument(
+          "The size of dimension about mean must equal to 1."
+          "But received: the shape of mean is [%s]"
+          "the size of dimension about mean is [%d]",
+          est_mean->dims(), est_mean->dims().size());
+      PADDLE_ENFORCE_EQ(est_var->dims().size(), 1UL,
+          platform::errors::InvalidArgument(
+          "The size of dimension about variance must equal to 1."
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cu]: dimension -》dimensions?

Comment (wanghaoshuang) [reply paddle/fluid/operators/batch_norm_op.cu]: dimension about variance?

File: paddle/fluid/operators/batch_norm_op.cu [hunk 4]
```diff
@@ -151,10 +154,29 @@ class BatchNormKernel<platform::CUDADeviceContext, T>
...
+          "But received: the shape of variance is [%s]"
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cu]: 应该先说dimensions' size = [%d]，然后再说dimensions是什么？另外dimensions和shape的定义是一样吧？都用dimensions是不是更合适？

File: paddle/fluid/operators/batch_norm_op.cu [hunk 5]
```diff
@@ -503,8 +525,11 @@ class BatchNormGradKernel<platform::CUDADeviceContext, T>
 
     const auto &x_dims = x->dims();
 
-    PADDLE_ENFORCE(x_dims.size() >= 2 && x_dims.size() <= 5,
-                   "The Input dim size should be between 2 and 5");
+    PADDLE_ENFORCE_EQ(x_dims.size() >= 2 && x_dims.size() <= 5, true,
+                      platform::errors::InvalidArgument(
+                          "The Input dim size should be between 2 and 5"
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cu]: dim不要用缩写

File: paddle/fluid/operators/batch_norm_op.cu [hunk 6]
```diff
@@ -503,8 +525,11 @@ class BatchNormGradKernel<platform::CUDADeviceContext, T>
...
+                          "But received: the dimension of Input is [%d]",
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cu]: 是不是把dimensions和dimension size弄错了？

File: paddle/fluid/operators/batch_norm_op.cu [hunk 7]
```diff
@@ -515,8 +540,18 @@ class BatchNormGradKernel<platform::CUDADeviceContext, T>
       d_scale->mutable_data<BatchNormParamType<T>>(ctx.GetPlace());
       d_bias->mutable_data<BatchNormParamType<T>>(ctx.GetPlace());
     }
-    PADDLE_ENFORCE_EQ(scale->dims().size(), 1UL);
-    PADDLE_ENFORCE_EQ(scale->dims()[0], C);
+    PADDLE_ENFORCE_EQ(
+        scale->dims().size(), 1UL,
+        platform::errors::InvalidArgument(
+            "The size of dimension about scale must equal to 1. But received: "
+            "the size of dimension about scale is [%d]",
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/batch_norm_op.cu]: about?

File: paddle/fluid/operators/instance_norm_op.cu [hunk 8]
```diff
@@ -257,8 +267,21 @@ class InstanceNormGradKernel<platform::CUDADeviceContext, T>
...
+                          "The `shape` in InstanceNormOp is invalid: "
+                          "the dimension of scale must be equal to 1. But "
+                          "received: the dimension of scale"
+                          "is [%d]",
+                          scale->dims().size()));
+    PADDLE_ENFORCE_EQ(scale->dims()[0], C,
+                      platform::errors::InvalidArgument(
+                          "The `shape` in InstanceNormOp is invalid: "
+                          "the first dimension of scale must be equal to "
+                          "Channels([%d]). But received: "
```
Comment (wanghaoshuang) [inline paddle/fluid/operators/instance_norm_op.cu:282]: 尽量避免'括号'这种表达方式

**Review by wanghaoshuang — CHANGES_REQUESTED** [review]

File: paddle/fluid/operators/batch_norm_op.cc (hunk 1 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cc]: done,thanks

File: paddle/fluid/operators/batch_norm_op.cc (hunk 2 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cc]: done,thanks

File: paddle/fluid/operators/batch_norm_op.cu (hunk 3 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cu]: done,thanks

File: paddle/fluid/operators/batch_norm_op.cu (hunk 4 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cu]: done,thanks

File: paddle/fluid/operators/instance_norm_op.cu (hunk 8 above)
Comment (ceci3) [reply paddle/fluid/operators/instance_norm_op.cu:282]: done,thanks

File: paddle/fluid/operators/batch_norm_op.cu (hunk 7 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cu]: done,thanks

File: paddle/fluid/operators/batch_norm_op.cu (hunk 6 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cu]: done,thanks

File: paddle/fluid/operators/batch_norm_op.cu (hunk 5 above)
Comment (ceci3) [reply paddle/fluid/operators/batch_norm_op.cu]: done,thanks

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**